### ISSUE TITLE
Is this project 100% developed with Claude?

### ISSUE THREAD

**Issue by sun-sun-sun** [ssochi/MacAIverse#1]
Is this project 100% developed with Claude? Is there any documentation or article on the development process that I can learn about?

**Comment by ssochi** [comment]
是的 1000%由Claude制作,我没有写过一个字符,当然我和Claude对话了无数轮.如果你想贡献代码可以参考我新提交的README_FOR_HUMAN.md

**Comment by ssochi** [comment]
Certainly, 1000% created by Claude, I haven't written a single character, but of course, I've had countless conversations with Claude. If you'd like to contribute code, you can refer to the newly submitted README_FOR_HUMAN.md that I committed.
